src/ad/abstract.js

```diff
@@ -0,0 +1,183 @@
+import { SortedListItem } from '../util/sorted-list-item'
+import { SortedList } from '../util/sorted-list'
+
+/**
+ * Abstract class for ads (i.e., {@link InLine} and {@link Wrapper}).
+ * @class AbstractAd
+ * @abstract
+ * @protected
+ * @extends SortedListItem
+ * @author Tim De Pauw <tim.depauw@zentrick.com>
```

Comment:
Feel free to either add yourself as an author or remove `@author` in favor of the `@copyright` that's already there. :slightly_smiling_face:

Reply:
I guess it's a bit clunky to have my name everywhere so rather than adding yours too, let's just remove all `@author`s. However, please add your name under "maintainers" in the readme and in package.json.